app/routes/claims-data.js

```diff
@@ -5,6 +5,16 @@ const crumbCache = require("./utils/crumb-cache");
 const { updateClaimData } = require("../api/claims");
 const { updateApplicationData } = require("../api/applications");
 
+const panelIdGivenFormName = (formName) => {
+  if (formName === "updateVetsName") {
+    return "#update-vets-name";
+  } else if (formName === "updateDateOfVisit") {
+    return "#update-date-of-visit";
+  } else {
+    return "#update-vet-rcvs-number";
+  }
+};
```

Comment:
```suggestion
const panelIdGivenFormName = (formName) => {
  if (formName === "updateVetsName") {
    return "#update-vets-name";
  }
  
  if (formName === "updateDateOfVisit") {
    return "#update-date-of-visit";
  }
  
   return "#update-vet-rcvs-number";
};
```

Reply:
Just a minor preference, I think these blocks are a little easier to read without the elses.